back/engines/commercial/idea_custom_fields/spec/acceptance/ideas_spec.rb

```diff
@@ -0,0 +1,82 @@
+require 'rails_helper'
+require 'rspec_api_documentation/dsl'
+
+resource 'Ideas' do
+  explanation 'Proposals from citizens to the city.'
+
+  let(:user) { create(:user) }
+
+  before do
+    header 'Content-Type', 'application/json'
+    @user = user
```

Comment:
Using instance variables in specs will not pass RuboCop. See https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/InstanceVariable

Please explain why you mix instance variables and `let`. I like to understand why you chose the former form instead of using `let` everywhere.